core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -922,6 +922,15 @@ private[spark] object SparkSubmitUtils {
 
         // A Module descriptor must be specified. Entries are dummy strings
         val md = getModuleDescriptor
+        // clear ivy resolution from previous launches. The resolution file is usually at
+        // ~/.ivy2/org.apache.spark-spark-submit-parent-default.xml. In between runs, this file
```

Comment:
@brkyvz, what determines the name of this file? I'm curious how often it's "usually" at this filename and whether it could sometimes be published at different names.